v2_1/ws/rest/docs/4_6_1_other_queries.md

```diff
@@ -0,0 +1,77 @@
+
+The following rules apply:
+
+- If no key is specified, it is equivalent to using the keyword `all`.
+- If no providerRef is specified, it equivalent to using the keyword `all`.
+- If no componentId is specified, it equivalent to using the keyword `all`.
+
+##### Parameters used to further filter the desired results
+
+The following query string parameters are used to further describe (or filter) the desired results, once the resource has been identified.
+
+Parameter | Type | Description | Default
+--- | --- | --- | ---
+startPeriod | common:StandardTimePeriodType, as defined in the SDMXCommon.xsd schema. Can be expressed using *dateTime* (all data that falls between the calendar dates will be matched), *Gregorian Period* (all data that falls between the calendar dates will be matched) or *Reporting Period* (all data reported as periods that fall between the specified periods will be returned. When comparing reporting weeks and days to higher order periods (e.g. quarters) one must account for the actual time frames covered by the periods to determine whether the data should be included. Data reported as Gregorian periods or distinct ranges will be returned if it falls between the specified reporting periods, based on a reporting year start day of January 1). In case the `:` or `+` characters are used, the parameter must be [percent-encoded](http://en.wikipedia.org/wiki/URL_encoding#Percent-encoding_reserved_characters) by the client. Note that this value is assumed to be inclusive to the range of data being sought. For additional information about the formats, see section 4.2.14 of Section 06 (SDMX Technical Notes). | The start period for which results should be supplied (inclusive). | 
+endPeriod | Same as above | The end period for which results should be supplied (inclusive). | 
+updatedAfter | xs:dateTime | The last time the query was performed by the client in the database. If this attribute is used, the returned message should only include the latest version of what has changed in the database since that point in time (updates and revisions). This should include observations that have been added since the last time the query was performed (INSERT), observations that have been revised since the last time the query was performed (UPDATE) and observations that have been deleted since the last time the query was performed (DELETE). If no offset is specified, default to local time of the web service. If the information about when the data has been updated is not available at the observation level, the web service should return either the series that have changed (if the information is attached at the series level) or the dataflows that have changed (if the information is attached at the dataflow level). | 
```

Comment:
This parameter changes the input data into the serivce's availability calculations. I find the text 'If this attribute is used, the returned message should only include the latest version of what has changed in the database since that point in time' a bit misleading.